broker/src/scheduler/active_calls_gc.erl

```diff
@@ -0,0 +1,46 @@
+% Active calls GC: Garbage collector for active calls.
+% When a call remains active for too long Verboice considers there was an error and cancels it.
+% This GC runs every INTERVAL. It cancels every active call for N (default=120) minutes.
+% This time (N) is configurable via the ENV variable minutes_for_cancelling_active_calls.
+-module(active_calls_gc).
+-export([code_change/3, handle_call/3, handle_cast/2, handle_info/2, init/1, start_link/0, terminate/2]).
+-compile([{parse_transform, lager_transform}]).
+-behaviour(gen_server).
+
+-define(SERVER, ?MODULE).
+
+-define(ONE_MINUTE, 60000).
+-define(TEN_MINUTES, 600000).
+
+-define(INIT_DELAY, ?ONE_MINUTE).
+-define(INTERVAL, ?TEN_MINUTES).
+
+start_link() ->
+  gen_server:start_link({local, ?SERVER}, ?MODULE, {}, []).
+
+init({}) ->
+  erlang:send_after(?INIT_DELAY,self(),cancel_active_calls),
+  {ok, undefined}.
+
+handle_call(_Request, _From, State) ->
+  {reply, {error, unknown_call}, State}.
+
+handle_cast(_Msg, State) ->
+  {noreply, State}.
+
+handle_info(cancel_active_calls, State) ->
+  N = verboice_config:minutes_for_cancelling_active_calls(),
+  Reason = "active_calls_gc N=" ++ integer_to_list(N),
```

Comment:
I'd rephrase this in a more user-friendly way. `active-too-long` or `kept-active` or `remained-active` or similar, and then add the details like `Call stayed active for more than N minutes`.